aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -307,6 +307,14 @@ template <class T> Vec256<T> inline operator*(const Vec256<T> &a, const Vec256<T
   return c;
 }
 
+template <> Vec256<bool> inline operator*(const Vec256<bool> &a, const Vec256<bool> &b) {
+  Vec256<bool> c = Vec256<bool>();
+  for (int i = 0; i != Vec256<bool>::size(); i++) {
+    c[i] = a[i] && b[i];
```

Comment:
Not sure if it is going to work in your case, but please consider checking https://scc.ustc.edu.cn/zlsc/sugon/intel/compiler_c/main_cls/index.htm#intref_cls/common/intref_avx_fmadd_pd.htm